index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Meeka's Personal Portfolio</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <header>
+            <nav> 
```

Comment:
appreciate these semantic tags here! see if there are any others you can use for other divs in the file: https://www.w3schools.com/html/html5_semantic_elements.asp